week-1-ts-ecosystem/w01e01-typescript-basics/src/main.ts

```diff
@@ -1,12 +1,8 @@
-/**
- * Zasady co do pliku:
- *
- * Możesz dowolnie modyfikować zawartość tego pliku,
- * całość programu musi jednak działać tak jak do tej pory !
- *
- * */
+import { hasAddress } from '../lib/has-address';
+import { hasGivenAge } from '../lib/has-given-age';
```

Comment:
To `../lib` chyba wyskoczyło Ci do innego folderu poza zadaniem :)`
w sensie jest w `w01e01`

Ja bym `lib` nazwał precyzyjniej jako `validators` - bo zobacz, że taką rolę pełnią te funkcje tutaj